### PR TITLE
Play Test

### PR DESCRIPTION
**Pull request by dylHall**
Play test me @PastaTime

### CODE REVIEW

File: ChainYoFriends/Assets/Prefabs/Bunny.prefab [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  m_Father: {fileID: 0}
+  m_RootOrder: 0
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!212 &2650626922375502587
+SpriteRenderer:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 2650626922375502583}
```
Comment (PastaTime) [inline ChainYoFriends/Assets/Prefabs/Bunny.prefab:43]: 2650626922375502584?

File: ChainYoFriends/ProjectSettings/TagManager.asset [hunk 2]
```diff
@@ -13,7 +13,7 @@ TagManager:
   - UI
   - 
   - 
-  - Player
+  - Bunny
```
Comment (PastaTime) [inline ChainYoFriends/ProjectSettings/TagManager.asset:16]: +Bunny

File: ChainYoFriends/Assets/Prefabs/Bunny.prefab (hunk 1 above)
Comment (dylHall) [reply ChainYoFriends/Assets/Prefabs/Bunny.prefab:43]: ah yeah will fix that next pull xd

**Comment by PastaTime** [comment]
LGTM

File: ChainYoFriends/ProjectSettings/TagManager.asset (hunk 2 above)
Comment (dylHall) [reply ChainYoFriends/ProjectSettings/TagManager.asset:16]: thanks

**Pull request merged**